app/code/Magento/Catalog/Console/Command/ImagesResizeCommand.php

```diff
@@ -95,15 +94,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $imageCache = $this->imageCacheFactory->create();
                 $imageCache->generate($product);
 
-                $output->write(".");
+                $this->productRepository->cleanCache();
```

Comment:
this method is out of RepositoryInterface, so we can not call it directly. As a workaround, we can check if method_exists and call it. In this case the fix will be BC